index.js

```diff
@@ -57,35 +57,40 @@ app.post('/api/setColumnMapping', function(req, res) {
 });
 
 // MANYC Sends to Group Object
-app.post('/api/createRequest', function(req, res) {
-  newRequest = {
-    manyc: {
-      status: req.body.status,
-      id: req.body.id,
-      supportType: req.body.supportType,
-      otherSupport: req.body.otherSupport,
-      community: req.body.community,
-      language: req.body.language,
-      languageOther: req.body.languageOther,
-      phone: req.body.phone,
-      email: req.body.email,
-      fullName: req.body.fullName,
-      urgency: req.body.urgency,
-      contactMethod: req.body.contactMethod,
-      crossStreet: req.body.crossStreet,
-      timestampCreated: req.body.timestampCreated,
-      timestampSent: req.body.timestampSent,
-      source: req.body.source,
-      sourceID: req.body.sourceID,
-    },
-  };
+app.post('/api/createRequest', async function(req, res) {
+  const newRequest = req.body;
+  // newRequest = {
+  //   manyc: {
```

Comment:
This whole part can be removed

Reply:
Take a look now, I also removed console.logs that are not necessary anymore.